index.js

```diff
@@ -5,15 +5,21 @@ const { GitHub, context } = require('@actions/github')
 
 const main = async () => {
   const token = core.getInput('github-token')
-  const branch = core.getInput('branch')
+  const head_branch = core.getInput('branch', { required: false }) || core.getInput('head_branch')
```

Comment:
```suggestion
  const branch = core.getInput('branch')
```

The argument pair `branch` and `base` seem fine too

Reply:
Also, in your PR, the variable `branch` is used but not defined. Accepting the suggested changes should fix that